Dockerfile-centos-7

```diff
@@ -3,10 +3,14 @@ FROM centos/systemd
 ARG FPM_VERSION=1.11.0
 ARG PYTHON_VERSION=3
 ARG DOCKER_WORKDIR=/usr/share/pdagent
+ARG SKIP_SYSTEMD=false
 ENV PYTHON_VERSION ${PYTHON_VERSION}
 ENV container docker
 ENV DOCKER_WORKDIR ${DOCKER_WORKDIR}
 
+# There is known issues with Docker + systemd
```

Comment:
Did we only want to include this for testing? Is it helpful for it to remain in the changes or at least not commented out for people who will inevitably build from source?

Reply:
ahhh disregard I hadn't seen you have a config boolean for this.